tests/validators/test_typed_dict.py

```diff
@@ -331,10 +332,17 @@ def test_validate_assignment_ignore_extra():
     assert v.validate_python({'field_a': 'test'}) == ({'field_a': 'test'}, {'field_a'})
 
     with pytest.raises(ValidationError) as exc_info:
-        v.validate_assignment('other_field', 456, {'field_a': 'test'})
+        v.validate_assignment({'field_a': 'test'}, 'other_field', 456)
 
+    # insert_assert(exc_info.value.errors())
     assert exc_info.value.errors() == [
-        {'type': 'extra_forbidden', 'loc': ('other_field',), 'msg': 'Extra inputs are not permitted', 'input': 456}
+        {
+            'type': 'no_such_attribute',
+            'loc': ('other_field',),
+            'msg': "Object has no attribute 'other_field'",
+            'input': 456,
+            'ctx': {'attribute': 'other_field'},
+        }
```

Comment:
I imagine that in `pydantic` we'll want to catch this and do something like `raise AttributeError(...) from e`?

Reply:
currently we don't but we could think about it.